modules/mongodb/templates/mongodb-restore-s3.erb

```diff
@@ -14,7 +14,7 @@ cd $BACKUP_DIR
 TIME="$(date +%s)"
 
 # Download and decrypt backup
-  /usr/bin/envdir <%= @env_dir %>/env.d /usr/local/bin/s3cmd get --force `/usr/local/bin/s3cmd ls s3://${S3_BUCKET}/mongodump* | \
+  /usr/bin/envdir <%= @env_dir %>/env.d /usr/local/bin/s3cmd get --force `/usr/bin/envdir /etc/mongo_s3backup/env.d /usr/local/bin/s3cmd ls s3://${S3_BUCKET}/mongodump* | \
```

Comment:
Is there a reason one of these uses `<%= @env_dir %>` and the other is the literal `/etc/mongo_s3backup/env.d`?